sessions/backend-mongodb/queries/README.md

```diff
@@ -0,0 +1,52 @@
+# Backend MongoDB: Queries
+
+In this challenge, you'll use MongoDB Compass to create a database and a collection of documents.
+
+## Tasks
+
+### Preparatory Steps
+
+- Open the MongoDB Compass app on your device
+- Create a connection to your local database (it should be `localhost:27017` by default).
+- Create a new database called _neuefische_.
+- Within your _neuefische_ database, create a collection called _students_.
+- ⬇️ [Download the provided students' data here](./resources.zip?raw=true).
+- Unpack the `resources.zip` and import the `data.json` file into your _students_ collection.
+
+✨ You now have a set of students' objects you can work with!
+
+### Queries
+
+Open the _mongosh_ at the bottom of MongoDB Compass.
```

Comment:
Should we include instructions for people using the terminal instead? Probably not, but I figured I might ask.